src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react';
+import { Loader } from '../../components/Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import PeopleTable from '../../components/PeopleTable';
+
+const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [err, setErr] = useState('');
```

Comment:
```suggestion
  const [errorMessage, setErrorMessage] = useState('');
```

Also you can create `enum` for all posible error messages